### PR TITLE
added tests for map_sequence animator and test for plot

### LINKED ISSUE #2296: Mapcube animation is not tested
#2295 points out the fact we don't have a test suite for the plotting capability of MapSequence.
That's not good.
So we need a MapSequence plot test.

https://github.com/sunpy/sunpy/blob/master/sunpy/map/mapsequence.py
https://github.com/sunpy/sunpy/blob/master/sunpy/map/tests/test_mapsequence.py

MapSequence  animation relies on https://github.com/sunpy/sunpy/blob/master/sunpy/visualization/animator/mapsequenceanimator.py 

This code has no tests - see https://github.com/sunpy/sunpy/tree/master/sunpy/visualization/animator/tests 

Tests for MapSequenceAnimator go in this subdirectory.
All the methods for the MapSequenceAnimator object will need tests.

A previous attempt was made here (when it was called MapCube): https://github.com/sunpy/sunpy/pull/2564

### PR DESCRIPTION
**Pull request by jeffreypaul15**
### Description

Fixes #2296 
Re-opened this PR for adding tests to `MapSequenceAnimator`, also added test for `MapSequence.plot()`.

### CODE REVIEW

**Comment by nabobalis** [comment]
You need to delete "sunpy/tests/figure_hashes_mpl_332_ft_261_astropy_403.json" and add the hashes to its replacement.

**Comment by jeffreypaul15** [comment]
> You need to delete "sunpy/tests/figure_hashes_mpl_332_ft_261_astropy_403.json" and add the hashes to its replacement.

Oops, yeah sorry about that.

File: changelog/5125.trivial.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Added tests for `sunpy.visualization.animator.mapsequenceanimator` and a test for using
+`plot()` with `sunpy.map.MapSequence`.
```
Comment (nabobalis) [inline changelog/5125.trivial.rst:2]:
```suggestion
Added tests for `sunpy.visualization.animator.mapsequenceanimator` and :meth:`sunpy.map.MapSequence.plot`.
```

File: sunpy/tests/figure_hashes_mpl_dev_ft_261_astropy_dev.json [hunk 2]
```diff
@@ -44,6 +45,10 @@
   "sunpy.timeseries.tests.test_timeseriesbase.test_noaa_txt_pre_peek": "fbda770f164bd38a7741d3393e9f93040b188e928521cc53379ba2b78beb582c",
   "sunpy.timeseries.tests.test_timeseriesbase.test_generic_ts_peek": "73a71e717b7dd0dd345f0f8a8bfa7f06387607417b3ecd123adfd8f630f93d51",
   "sunpy.visualization.animator.tests.test_basefuncanimator.test_lineanimator_figure": "dbf47ab983a844621f912b2f5a2d9ee657d3f661678c16a07afc7217f50e3fd4",
+  "sunpy.visualization.animator.tests.test_mapsequence.test_map_sequence_animator_wcs_simple_plot": "0b4d489854b85b6fe469c61c8b1920183e06bb59f2075630d23f46a3c516e030",
+  "sunpy.visualization.animator.tests.test_mapsequence.test_map_sequence_animator_wcs_colorbar_buttons": "fcd5d64e48e45c3eebe73a835a22921bfb80b5a8b63eb85faddb68a2cfb447d1",
+  "sunpy.visualization.animator.tests.test_mapsequence.test_map_sequence_animator_wcs_colorbar_buttons_default_labels": "efc609d0263597f1c66ff578262f14a2e1cee73bf90e2fb2f44334d82fed85f6",
+  "sunpy.visualization.animator.tests.test_mapsequenceanimator.test_map_sequence_animator_wcs_simple_plot": "0b4d489854b85b6fe469c61c8b1920183e06bb59f2075630d23f46a3c516e030",
```
Comment (nabobalis) [inline sunpy/tests/figure_hashes_mpl_dev_ft_261_astropy_dev.json:51]: Do all of these need to be here?

**Comment by jeffreypaul15** [comment]
@nabobalis I don't suppose the failing test is due to the code change here right?

**Comment by nabobalis** [comment]
No they are unrelated to your changes.

File: sunpy/tests/figure_hashes_mpl_dev_ft_261_astropy_dev.json [hunk 3]
```diff
@@ -44,6 +45,10 @@
   "sunpy.timeseries.tests.test_timeseriesbase.test_noaa_txt_pre_peek": "fbda770f164bd38a7741d3393e9f93040b188e928521cc53379ba2b78beb582c",
   "sunpy.timeseries.tests.test_timeseriesbase.test_generic_ts_peek": "73a71e717b7dd0dd345f0f8a8bfa7f06387607417b3ecd123adfd8f630f93d51",
   "sunpy.visualization.animator.tests.test_basefuncanimator.test_lineanimator_figure": "dbf47ab983a844621f912b2f5a2d9ee657d3f661678c16a07afc7217f50e3fd4",
+  "sunpy.visualization.animator.tests.test_mapsequence.test_map_sequence_animator_wcs_simple_plot": "0b4d489854b85b6fe469c61c8b1920183e06bb59f2075630d23f46a3c516e030",
+  "sunpy.visualization.animator.tests.test_mapsequence.test_map_sequence_animator_wcs_colorbar_buttons": "fcd5d64e48e45c3eebe73a835a22921bfb80b5a8b63eb85faddb68a2cfb447d1",
+  "sunpy.visualization.animator.tests.test_mapsequence.test_map_sequence_animator_wcs_colorbar_buttons_default_labels": "efc609d0263597f1c66ff578262f14a2e1cee73bf90e2fb2f44334d82fed85f6",
```
Comment (nabobalis) [inline sunpy/tests/figure_hashes_mpl_dev_ft_261_astropy_dev.json:50]:
```suggestion
```

**Comment by nabobalis** [comment]
Thanks for the PR @jeffreypaul15

**Review by nabobalis — APPROVED** [review]

**Pull request merged**